third_party/txt/src/txt/platform_windows.cc

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "third_party/skia/include/ports/SkTypeface_win.h"
+#include "txt/platform.h"
+
+namespace txt {
+
+std::string GetDefaultFontFamily() {
+  return "Arial";
```

Comment:
This change seems unrelated. Is this improving font fallback handling? If so, it would be great to split this out into its own change as part of #39915

Also, I'm surprised to see a file being added to third_party. What is the source for `txt`, and why isn't this an upstream change?

Reply:
This is essential for reloading system fonts. The original GetDefaultFontFamily in third_party/txt/src/txt/platform.cc uses SkFontMgr::RefDefault() which will cache the result and there is no way to clear it. When a new system font is installed, it will not load the latest system fonts but return previous cached. We have to implement the windows version of it.

There is no upstream for txt. The txt's source lives in here. One thing that worries me is that this might be migrated to skia library at some point @jason-simmons Do you think it is ok to change it now?

Reply:
Yes - go ahead and modify this in libtxt.  if we later migrate to Skia's text shaper then we will need to port this feature over to the integration with Skia.